File: Kattis/judging-moose.py
left, right = map(int, input().split())

if left == right == 0:
    print("Not a moose")
elif left == right:
    total = str(left + right)
    print("Even " + total)
elif left > right:
    total = str(2 * left)
    print("Odd " + total)
else:
    total = str(2 * right)
    print("Odd " + total)

File: Kattis/C/judging_moose.c
#include <stdio.h>

int main(void) {
    int left, right;
    scanf("%d %d", &left, &right);
    if (left == 0 && right == 0) {
        printf("Not a moose");
    } else if (left == right) {
        printf("Even %d", left + right);
    } else if (left > right) {
        printf("Odd %d", 2 * left);
    } else {
        printf("Odd %d", 2 * right);
    }
    return 0;
}
